### PR TITLE
Add missing translations on views

### PR DESCRIPTION
**Pull request by pacodelaluna**

### CODE REVIEW

**Comment by pacodelaluna** [comment]
@mkllnk Related to the issue https://github.com/openfoodfoundation/openfoodnetwork/issues/1095, please let me know if any issue.

File: config/locales/fr.yml [hunk 1]
```diff
@@ -475,9 +475,9 @@ fr:
         title: Paramètres de facturation
         invoice_style2?: Utiliser le modèle de facture alternatif qui détaille le montant de TVA agrégé par taux et l'information du taux de TVA par produit (pas adapté pour les instances affichant les prix HT)
         enable_receipt_printing?: Afficher les options d'impression de tickets de caisse dans le menu déroulant des commandes?
-  home:
+  shops:
```
Comment (pacodelaluna) [inline config/locales/fr.yml]: No worries, I will also update the translations on Transifex once the commit is merged to avoid any regression.

**Comment by daniellemoorhead** [comment]
Hi @pacodelaluna, @mkllnk hasn't been doing OFN things this week, he'll be back next Wednesday though so we should be able to make this a priority then. Cheers!

File: config/locales/fr.yml [hunk 2]
```diff
@@ -516,9 +516,9 @@ fr:
     already_ordered:
       cart: "panier"
       message_html: "Vous avez déjà passé une commande pour ce cycle de vente. Vérifiez votre %{cart} pour voir les produits commandés. Vous pouvez annuler ou modifier votre commande jusqu'à la fermeture du cycle de vente."
-  home:
+  shops:
```
Comment (mkllnk) [inline config/locales/fr.yml]: I hope this doesn't cause a conflict when merging changes from Transifex after the next release. Let's see...

Comment (enricostano) [inline config/locales/fr.yml]: I was about to comment the same thing, we should only commit `en.yml` file and manage the rest through Transifex process 🙏

Comment (sauloperez) [inline config/locales/fr.yml]: We must add this then at the top-level documentation of the `en.yml` that @oeoeaio added. It gives some guidelines when dealing with translations in OFN.

Comment (enricostano) [inline config/locales/fr.yml]: This is a good example where the Code Climate engines could come handy, the [grep](https://docs.codeclimate.com/docs/grep) one for instance.

**Comment by sstead** [comment]
@mkllnk I've checked the pages listed and could not see any translation issues. 

Note I saw this again, but it was already spotted in previous PR...
image

This is probably not related but on the shops page only some category icons were showing...
image.

**Comment by mkllnk** [comment]
I think that these two issues are unrelated. I'm merging it.

File: config/locales/fr.yml (hunk 2 above)
Comment (mkllnk) [inline config/locales/fr.yml]: I thought about this a bit more and it is very unlikely to cause a conflict. But Transifex will overwrite `fr.yml` once it's translated. Just when we get new translations from Transifex and have changes in the master branch at the same position of the locale, we can get merge conflicts.

**Pull request merged**